module#2/src/api/controllers/users-controllers.ts

```diff
@@ -0,0 +1,83 @@
+import { Request, Response } from "express";
+import { dataBase, ERROR_MESSAGE, HTTP_STATUSES } from "../../constants";
+import { v4 as id } from "uuid";
+import { getAutoSuggestUsers, isUserAlreadyExist } from "../../utils/user-operations";
+
+export const getAllUsers = (req: Request, res: Response) => {
```

Comment:
Excuse me, where do we apply sorting by login property?

Reply:
Oh, sorry for confusing. I forgot about it. Added. Thanks a lot)